CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.12.0] - released on 2017-05-31
+### Added
+- Added `generate_source_artifacts` method to the platform dsl to allow generation
+  of `.src.rpm` and debian source artifacts.
+- Added the ability to specify mirror URLs.
+
+### Deprecated
+- The existing rewrite engine is being deprecated in favor of the mirror URLs.
```

Comment:
I'd probably expand this out a little bit more; it's a meaningful change for users.